components/post_view/post/post.jsx

```diff
@@ -106,12 +106,13 @@ export default class Post extends React.PureComponent {
     }
 
     handleCommentClick = (e) => {
+        const post = this.props.post || {};
```

Comment:
What's the point of dispatching the action if the post is `{}`? Why not just not dispatch.